### PR TITLE
CMCL-983: Add Input system projects

### PR DESCRIPTION
**Pull request by AntoineCharton**
### Purpose of this PR

Add input system test projects that will run on nightly tests. 

### Testing status

Add more coverage to projects testing. Also test it against the different render pipelines HDRP and URP.

### Technical risk

Low infrastructure and testing changes

### CODE REVIEW

File: .yamato/metadata.metafile [hunk 1]
```diff
@@ -38,10 +38,19 @@ all_testprojects:
...
+      path: Projects/StandaloneInputSystem
+      editors: [2022.2, trunk]
+    - name: HDRPInputSystem
+      path: Projects/HDRPInputSystem
+      editors: [2022.2, trunk]
 
 all_configurations:
   ci:
-    test_names: [isolation_tests]
+    test_names: [isolation_tests, promotion_tests]
```
Comment (gaborkb) [inline .yamato/metadata.metafile:53]: I think @guillaumelevass removed promotion tests for a reason. I might be mistaken :)

Comment (gaborkb) [reply .yamato/metadata.metafile:53]: I checked git history and that's not the case. @guillaumelevass did not remove this. He add promotion tests to the nightly only, so I remembered wrong. :)

Why did you add it?

Comment (AntoineCharton) [reply .yamato/metadata.metafile]: Fixed. Was just a test.

File: Projects/HDRPInputSystem/UserSettings/Search.settings [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (gaborkb) [inline Projects/HDRPInputSystem/UserSettings/Search.settings:1]: Is this file needed?

**Review by gaborkb — COMMENTED** [review]
Started the nightly tests to check that everything is alright. Will approve after they pass.
https://unity-ci.cds.internal.unity3d.com/job/18806514/dependency-graph

**Review by gaborkb — APPROVED** [review]
Failed errors are just package version related:
Error: This package iteration (3) must be higher than the highest published iteration (3).

**Review by guillaumelevass — APPROVED** [review]
